data/images/README.md

```diff
@@ -1,3 +1,2 @@
 # Directory for profile images
 
-If you are a volunteer for The Laurel: add your profile image .png with the name being your Rinkeby Eth address that you use for the system. Then do a PR.
```

Comment:
1. This text needs to be put back in the `data/images/README.md` file

Reply:
2. Your image file `data/images/0xB71B6F11c72EeB4aFb45D2e3b7057b742a5fb486.png.png` needs to be named `data/images/0xB71B6F11c72EeB4aFb45D2e3b7057b742a5fb486.png` (automation reasons).